client/gutenberg/extensions/videopress/editor.js

```diff
@@ -43,8 +47,40 @@ const addVideoPressSupport = ( settings, name ) => {
 				default: 'metadata',
 			},
 		},
+
+		transforms: {
+			...settings.transforms,
+			from: [
+				{
+					type: 'files',
+					isMatch( files ) {
+						return files.length === 1 && files[ 0 ].type.indexOf( 'video/' ) === 0;
+					},
+					// We define a higher priority (lower number) than the default of 10. This ensures that this
+					// transformation prevails over the core video block default transformations.
+					priority: 9,
+					transform( files, onChange ) {
+						const file = files[ 0 ];
```

Comment:
What happens with multiple files — could we `filter()` + loop them all here?

Reply:
Good idea! I applied it on 61264f9 but only if all the dropped files are videos. 

Since Gutenberg only applies the first transformation that matches, I think it's safer to don't create blocks at all if the file types are different. Otherwise, we would be creating a video block when the user drops an image and a video and maybe they wanted to drop only an image (but accidentally dropped a video too).